### PR TITLE
reword explanation of how dirty bitmaps are returned for clarity

### PR DESCRIPTION
**Pull request by tmakatos**
Signed-off-by: Thanos Makatos <thanos.makatos@nutanix.com>

### CODE REVIEW

**Comment by tmakatos** [comment]
I'm sure we can explain it better, I'm open to suggestions.

**Review by jlevon — APPROVED** [review]
looks fine to me

File: docs/vfio-user.rst [hunk 1]
```diff
@@ -682,21 +688,18 @@ VFIO bitmap format
 The VFIO bitmap structure is defined in ``<linux/vfio.h>``
 (``struct vfio_bitmap``).
 
+Each ``struct_vfio_bitmap`` entry is followed by the region's bitmap. Each bit
```
Comment (swapnili) [inline docs/vfio-user.rst:691]: nit: struct vfio_bitmap

**Review by swapnili — APPROVED** [review]

File: docs/vfio-user.rst (hunk 1 above)
Comment (tmakatos) [reply docs/vfio-user.rst]: fixed

**Pull request merged**